tests/e2e/default/wdio.conf.js

```diff
@@ -1,15 +1,16 @@
 const allure = require('allure-commandline');
 const fs = require('fs');
-const constants = require('./constants');
-const utils = require('./utils');
+const constants = require('../../constants');
+const utils = require('../../utils');
 const path = require('path');
 
 const chai = require('chai');
 chai.use(require('chai-exclude'));
 
 const ALLURE_OUTPUT = 'allure-results';
-const browserLogPath = path.join(__dirname, 'logs', 'browser.console.log');
-const browserUtils = require('./utils/browser');
+//const browserLogPath = path.join(__dirname, 'logs', 'browser.console.log');
```

Comment:
I think we can remove this comment, we can alway see the commit history if needed.